coucharchive

```diff
@@ -447,6 +450,10 @@ def main():
     sub['restore'].add_argument(
         '-i', '--input', dest='input', action='store', required=True,
         help='path to archive to restore')
+    sub['restore'].add_argument(
+        '--force-continue', dest='force_continue',
+        action='store_true', default=False,
+        help='force restoration even if file exists on target')
```

Comment:
`file exists` -> not very clear with `databases` and `documents`

`continue restoration even if database exists on target`?

Reply:
Indeed! I'll update the text with yours.